public/app/features/trails/Integrations/SceneDrawer.tsx

```diff
@@ -17,6 +18,21 @@ export function SceneDrawer(props: SceneDrawerProps) {
   const { scene, title, onDismiss } = props;
   const styles = useStyles2(getStyles);
 
+  useEffect(() => {
+    // Adding this additional push prevents an embed in scenes dashboards from going back too far
+    locationService.push(locationService.getLocation());
```

Comment:
this feels a bit too hacky, if you open the drawer and then close it you have just an extra identical history item.

think to fix this we should make the drawer being open be in URL state (like other drawers are in dashboards, inspect drawer for example). Anyway I don't think we need to fix this issue right now, the drawer closes anyway (at least in scenes version) when you move to a different route/page

and after https://github.com/grafana/grafana/pull/84916 it will close in old architecture as well

Reply:
Ok, I will omit this from this PR.